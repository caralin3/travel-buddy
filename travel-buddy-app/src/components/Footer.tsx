import React from 'react';
import { Col, Row } from 'reactstrap';

export const Footer: React.FC = () => (
  <footer className="bg-light p-2">
    <Row>
      <Col>Travel Buddy</Col>
    </Row>
    <Row>
      <Col>
        <small className="text-muted">Copyright Â© 2022</small>
      </Col>
    </Row>
  </footer>
);
