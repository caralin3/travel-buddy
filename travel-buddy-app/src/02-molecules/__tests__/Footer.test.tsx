import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { Footer } from '../Footer';

describe('Footer', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<Footer />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
