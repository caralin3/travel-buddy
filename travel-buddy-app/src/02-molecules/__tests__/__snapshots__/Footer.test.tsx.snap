// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render correctly 1`] = `
<footer
  className="bg-light p-2"
>
  <Row
    tag="div"
    widths={
      Array [
        "xs",
        "sm",
        "md",
        "lg",
        "xl",
        "xxl",
      ]
    }
  >
    <Col
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
          "xxl",
        ]
      }
    >
      Travel Buddy
    </Col>
  </Row>
  <Row
    tag="div"
    widths={
      Array [
        "xs",
        "sm",
        "md",
        "lg",
        "xl",
        "xxl",
      ]
    }
  >
    <Col
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
          "xxl",
        ]
      }
    >
      <small
        className="text-muted"
      >
        Copyright Â© 2022
      </small>
    </Col>
  </Row>
</footer>
`;
